app/ui.js

```diff
@@ -965,9 +983,171 @@ var UI;
 /* ------^-------
  *     /XVP
  * ==============
- *   CLIPBOARD
+ *    AST2100
  * ------v------*/
 
+        ast2100_handleVideoSettingsChanged: function (settings) {
+            if (!UI._ast2100_videoSettingsInitialized)
+                UI.ast2100_setDefaultSettings(settings);
+            UI.ast2100_updateVideoSettings(settings);
+        },
+        
+        // Called the first time we receive a FramebufferUpdate object.  Responsible
+        // for telling the server about any configured default settings.
+        ast2100_setDefaultSettings: function (settings) {
+            // Convert the settings that noVNC has been configured to set for all
+            // AST2100 servers to the familiar videoSettings format.
+            var defaultQuality = parseInt(UI.ast2100_quality);
+            var defaultSettings = {
+                quantTableSelectorLuma: defaultQuality,
+                quantTableSelectorChroma: defaultQuality,
+                subsamplingMode: parseInt(UI.ast2100_subsamplingMode)
```

Comment:
Currently these are hard coded. Probably because no-one has bothered adding a UI for it. :)

You have these settings:

https://github.com/rfbproto/rfbproto/blob/master/rfbproto.rst#jpeg-quality-level-pseudo-encoding
https://github.com/rfbproto/rfbproto/blob/master/rfbproto.rst#jpeg-fine-grained-quality-level-pseudo-encoding
https://github.com/rfbproto/rfbproto/blob/master/rfbproto.rst#jpeg-subsampling-level-pseudo-encoding